### PR TITLE
chore: update driver adapter READMEs

### PR DESCRIPTION
**Pull request by ruheni**
Closes https://github.com/prisma/team-orm/issues/473

- [x] PlanetScale
- [x] Neon 
- [x] Turso

### CODE REVIEW

File: packages/adapter-neon/README.md [hunk 1]
```diff
@@ -1,29 +1,14 @@
-# @prisma/adapter-neon
+# Prisma driver adapter for Neon
```
Comment (janpio) [inline packages/adapter-neon/README.md:1]: Neon serverless driver

File: packages/adapter-planetscale/README.md [hunk 2]
```diff
@@ -1,29 +1,14 @@
-# @prisma/adapter-planetscale
+# Prisma driver adapter for PlanetScale@prisma/adapter-planetscale
```
Comment (janpio) [inline packages/adapter-planetscale/README.md:1]: mistake

+ Planetscale serverless driver

File: packages/adapter-planetscale/README.md [hunk 3]
```diff
@@ -33,24 +18,36 @@ generator client {
...
+  url      = env("DATABASE_URL")
 }
 ```
 
-Now run `npx prisma generate` to re-generate Prisma Client.
+> **Note**: Ensure you update the host value in your connection string to `aws.connect.psdb.cloud`. You can learn more about this [here](https://planetscale.com/docs/tutorials/planetscale-serverless-driver#add-and-use-the-planetscale-serverless-driver-for-javascript-to-your-project).
+>
+>```bash
+>DATABASE_URL='mysql://johndoe:strongpassword@aws.connect.psdb.cloud/clear_nightsky?sslaccept=strict'
+>```
```
Comment (janpio) [inline packages/adapter-planetscale/README.md:30]:
```suggestion
> ```bash
> DATABASE_URL='mysql://johndoe:strongpassword@aws.connect.psdb.cloud/clear_nightsky?sslaccept=strict'
> ```
```

File: packages/adapter-planetscale/README.md [hunk 4]
```diff
@@ -33,24 +18,36 @@ generator client {
...
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
 }
 ```
 
-Now run `npx prisma generate` to re-generate Prisma Client.
+> **Note**: Ensure you update the host value in your connection string to `aws.connect.psdb.cloud`. You can learn more about this [here](https://planetscale.com/docs/tutorials/planetscale-serverless-driver#add-and-use-the-planetscale-serverless-driver-for-javascript-to-your-project).
+>
+>```bash
+>DATABASE_URL='mysql://johndoe:strongpassword@aws.connect.psdb.cloud/clear_nightsky?sslaccept=strict'
```
Comment (janpio) [inline packages/adapter-planetscale/README.md:29]: Do single quote actually work here?

**Review by Jolg42 — COMMENTED** [review]
General question: this might get out of sync with docs regularly, keeping the READMEs tiny and pointing to docs for instructions would work better?

File: packages/adapter-planetscale/README.md (hunk 4 above)
Comment (ruheni) [reply packages/adapter-planetscale/README.md]: Unsure, I didn't test it out.

**Comment by ruheni** [comment]
> General question: this might get out of sync with docs regularly, keeping the READMEs tiny and pointing to docs for instructions would work better?

I think the PlanetScale and Neon adapter READMEs should be fine. We can trim down the content on the libSQL adapter README. WDYT?  🤔

File: packages/adapter-neon/README.md [hunk 5]
```diff
@@ -38,18 +23,23 @@ datasource db {
...
 
-Now run `npx prisma generate` to re-generate Prisma Client.
+Generate Prisma Client:
 
-## How to use
+```sh
+npx prisma generate
+```
+
+Install the Prisma adapter for Neon's serverless driver, Neon's serverless driver and `ws`:
```
Comment (Jolg42) [inline packages/adapter-neon/README.md:32]:
```suggestion
Install the Prisma adapter for Neon's serverless driver, Neon's serverless driver and `ws` packages:
```

**Review by Jolg42 — APPROVED** [review]
Nice ✨

**Comment by Jolg42** [comment]
> > General question: this might get out of sync with docs regularly, keeping the READMEs tiny and pointing to docs for instructions would work better?
> 
> I think the PlanetScale and Neon adapter READMEs should be fine. We can trim down the content on the libSQL adapter README. WDYT? 🤔

Just asking, I thought we were more on a single source of truth approach in the past to avoid extra work to keep things in sync.

File: packages/adapter-neon/README.md [hunk 6]
```diff
@@ -1,29 +1,14 @@
...
+# Prisma driver adapter for Neon serverless driver
 
-Prisma driver adapter for [Neon Serverless Driver](https://github.com/neondatabase/serverless).
+Prisma driver adapter for [Neon Serverless Driver](https://github.com/neondatabase/serverless). Refer to the [announcement blog post](https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV) and our [docs](https://www.prisma.io/docs/guides/database/neon#how-to-use-neons-serverless-driver-with-prisma-preview) for more details.
 
-See https://github.com/prisma/prisma/releases/tag/5.4.0 and https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV for details.
+> **Note:**: Support for Neon's serverless driver is available from Prisma versions [5.4.2](https://github.com/prisma/prisma/releases/tag/5.4.0) and later.
 
-The following usage tutorial is valid for Prisma 5.4.2 and later versions.
+The [Neon serverless driver](https://github.com/neondatabase/serverless) is a low-latency Postgres driver for JavaScript and TypeScript that allows you to query data from serverless and edge environments over HTTP or WebSockets.
```
Comment (jkomyno) [inline packages/adapter-neon/README.md:7]: I'm not sure I would mention `HTTP` here, considering how little support Prisma can provide for it

**Review by jkomyno — APPROVED** [review]
Approving this despite a few open comments, but let's be mindful that, having no references to Neon and PlanetScale databases to Prisma anymore, users that haven’t opened the blog posts won’t be able to follow up independently, by reading just the READMEs. If we're fine with this, than let's proceed. Otherwise, let's include back sections similar to:

```markdown
Make sure your [Neon database connection string](https://neon.tech/docs/connect/connect-from-any-app) is copied over to your `.env` file. The connection string will start with `postgres://`.
```

Also, tiny nit: I would keep references to the package names we decided to publish the Driver Adapters to `npm` with (e.g., `@prisma/adapter-neon`).

File: packages/adapter-neon/README.md [hunk 7]
```diff
@@ -1,29 +1,14 @@
-# @prisma/adapter-neon
+# Prisma driver adapter for Neon serverless driver
 
-Prisma driver adapter for [Neon Serverless Driver](https://github.com/neondatabase/serverless).
+Prisma driver adapter for [Neon Serverless Driver](https://github.com/neondatabase/serverless). Refer to the [announcement blog post](https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV) and our [docs](https://www.prisma.io/docs/guides/database/neon#how-to-use-neons-serverless-driver-with-prisma-preview) for more details.
 
-See https://github.com/prisma/prisma/releases/tag/5.4.0 and https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV for details.
+> **Note:**: Support for Neon's serverless driver is available from Prisma versions [5.4.2](https://github.com/prisma/prisma/releases/tag/5.4.0) and later.
```
[the remark this answers is not in the document]
Comment (ruheni) [reply packages/adapter-neon/README.md:5]: Yes. I wanted to link to the release notes where we launched it but it's a good call out 😄

File: packages/adapter-neon/README.md (hunk 6 above)
Comment (ruheni) [reply packages/adapter-neon/README.md:7]: I can omit both HTTP and WebSockets in the meantime 🙂

File: packages/adapter-planetscale/README.md [hunk 8]
```diff
@@ -33,24 +18,36 @@ generator client {
...
+  url      = env("DATABASE_URL")
 }
 ```
 
-Now run `npx prisma generate` to re-generate Prisma Client.
+> **Note**: Ensure you update the host value in your connection string to `aws.connect.psdb.cloud`. You can learn more about this [here](https://planetscale.com/docs/tutorials/planetscale-serverless-driver#add-and-use-the-planetscale-serverless-driver-for-javascript-to-your-project).
+>
+> ```bash
+> DATABASE_URL="mysql://johndoe:strongpassword@aws.connect.psdb.cloud/clear_nightsky?sslaccept=strict"
+> ```
```
Comment (jkomyno) [inline packages/adapter-planetscale/README.md:30]:
```suggestion
> DATABASE_URL="mysql://user:password@aws.connect.psdb.cloud/database_name?sslaccept=strict"
> ```
```

File: packages/adapter-planetscale/README.md [hunk 9]
```diff
@@ -33,24 +18,36 @@ generator client {
 }
 
 datasource db {
-  provider     = "mysql"
-  url          = env("DATABASE_URL")
-  relationMode = "prisma"
+  provider = "mysql"
+  url      = env("DATABASE_URL")
```
Comment (jkomyno) [inline packages/adapter-planetscale/README.md:22]:
```suggestion
  provider     = "mysql"
  url          = env("DATABASE_URL")
  relationMode = "prisma"
```

File: packages/adapter-neon/README.md [hunk 10]
```diff
@@ -38,18 +23,29 @@ datasource db {
 }
 ```
 
-Now run `npx prisma generate` to re-generate Prisma Client.
+> **Note**: Make sure your connection string exists in your `.env` file. Refer to [Neon's docs](https://neon.tech/docs/connect/connect-from-any-app) to learn how to retrieve your database's connection string.
+>
+> ```bash
+> DATABASE_URL="postgres://daniel:<password>@ep-mute-rain-952417.us-east-2.aws.neon.tech/neondb"
+> ```
```
Comment (jkomyno) [inline packages/adapter-neon/README.md:30]:
```suggestion
> ```bash
> DATABASE_URL="postgres://user:password@server.us-east-2.aws.neon.tech/neondb"
> ```
```

**Pull request merged**